GOVERNANCE.md

```diff
@@ -0,0 +1,21 @@
+# Governance
+
+## Committers
+
+Committers are contributors who have made several valuable contributions to the project and have earned the ability to modify (“commit”) source code, documentation or other technical artifacts in the project’s repositories. Besides contributing to the project, committers are relied on to review and validate the contributions of others. In many cases they are programmers but it is also possible that they contribute in a different role. Typically, a committer will focus on a specific aspect of the project, and will bring a level of expertise and understanding that earns them the respect of the community and the project owner.
+
+
+## Contributors
+
+Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project.
+
+Anyone can become a contributor. There is no expectation of commitment to the project, no specific skill requirements and no selection process. To become a contributor, a community member simply has to perform one or more actions that are beneficial to the project.
+
+## Responsibilities
+
+The Dynaw&omega;o project is split into several [repositories](https://github.com/dynawo). There is at least one committer in charge of each repository. By “in charge”, we mean:
+* Best effort to review the pull request,
+* Best effort to resolve issues,
+* Building and publishing the releases, including writing the release notes and informing the community
+
+Please refer to our maintainers list for more details about our work division.
```

Comment:
We should then specify in the codeowners that we give the committers list?

Reply:
just a sentence I have started and forgot to remove